### PR TITLE
performance: Don't fetch version for absolute versions

### PR DESCRIPTION
**Pull request by DeMoorJasper**
## What kind of change does this PR introduce?

If a version is absolute we don't need to do an additional resolve, this PR returns early in case of absolute versions.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6276/commits/4f32c6c4069fbdff61e9134848d18161cfaa218b failed.

File: packages/common/src/utils/dependencies.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { isAbsoluteVersion } from './dependencies';
+
+describe('dependencies', () => {
+  describe('isAbsoluteVersion', () => {
+    it('returns true on absolute versions', () => {
+      expect(isAbsoluteVersion('1.1.0')).toBe(true);
+      expect(isAbsoluteVersion('154.78.7896')).toBe(true);
+      expect(isAbsoluteVersion('1.1.0-next.1')).toBe(true);
+      expect(isAbsoluteVersion('154.78.123-alpha.1-235f8723g5oi23')).toBe(true);
```
Comment (CompuIves) [inline packages/common/src/utils/dependencies.test.ts:9]: I would also add another check here for `codesandbox/sandpack`

Comment (CompuIves) [reply packages/common/src/utils/dependencies.test.ts:9]: GitHub versions!

**Review by CompuIves — APPROVED** [review]
Nice improvements! I would make one small change, then ready.

File: packages/common/src/utils/dependencies.ts [hunk 2]
```diff
@@ -33,28 +33,26 @@ async function fetchAllVersions(dep: string): Promise<JsDelivrApiResult> {
...
+const resolveVersionFromUnpkg = (
+  dep: string,
+  version: string
+): Promise<string> => {
+  return fetchWithRetries(
+    `https://unpkg.com/${dep}@${encodeURIComponent(version)}/package.json`
+  ).then(x => x.version);
+};
+
 export async function getLatestVersion(
```
Comment (CompuIves) [inline packages/common/src/utils/dependencies.ts:46]: I triple checked, I think this function is nowhere else used. We could remove the `export` here.
```suggestion
async function getLatestVersion(
```

**Pull request merged**